services/QuillLMS/app/models/activity.rb

```diff
@@ -254,8 +254,16 @@ def data_as_json
     data
   end
 
+  # translatable
   def self.translatable_field_name = 'landingPageHtml'
 
+  def translated_json(options = {})
```

Comment:
This is purely stylistic, but I wonder if this should be called `json_with_translations` or something, to make it clear that it's additive?  That's just a naming thing, though, so feel free to keep the existing name, too.

Reply:
the only thing is that this is also part of the `translatable` module. I like the new name though! I think I'll change it in a later PR when I switch up the API for the concept_feedback.